### ISSUE TITLE
install errors on itop 3.0.0-beta

### ISSUE THREAD

**Issue by efarias** [TeemIp/teemip-core-ip-mgmt#11]
I got this errors when trying to install on itop 3.0.0-beta

```
[Wed Nov 03 20:37:43.969012 2021] [php7:warn] [pid 582] [client 172.17.0.1:55214] PHP Warning:  Declaration of TeemIp\\TeemIp\\Extension\\IPv6Management\\Model\\_IPv6Subnet::GetName() should be compatible with DBObject::GetName($sType = Combodo\\iTop\\Core\\MetaModel\\FriendlyNameType::SHORT) in /var/www/html/env-production/teemip-ipv6-mgmt/src/Model/_IPv6Subnet.php on line 59, referer: http://localhost:8001/setup/wizard.php
[Wed Nov 03 20:37:43.970462 2021] [php7:warn] [pid 582] [client 172.17.0.1:55214] PHP Warning:  Declaration of TeemIp\\TeemIp\\Extension\\IPv6Management\\Model\\_IPv6Address::GetName() should be compatible with DBObject::GetName($sType = Combodo\\iTop\\Core\\MetaModel\\FriendlyNameType::SHORT) in /var/www/html/env-production/teemip-ipv6-mgmt/src/Model/_IPv6Address.php on line 32, referer: http://localhost:8001/setup/wizard.php
[Wed Nov 03 20:37:47.592549 2021] [php7:warn] [pid 582] [client 172.17.0.1:55214] PHP Warning:  Declaration of TeemIp\\TeemIp\\Extension\\IPv6Management\\Model\\_IPv6Subnet::GetName() should be compatible with DBObject::GetName($sType = Combodo\\iTop\\Core\\MetaModel\\FriendlyNameType::SHORT) in /var/www/html/env-production/teemip-ipv6-mgmt/src/Model/_IPv6Subnet.php on line 59, referer: http://localhost:8001/setup/wizard.php
[Wed Nov 03 20:37:47.596176 2021] [php7:warn] [pid 582] [client 172.17.0.1:55214] PHP Warning:  Declaration of TeemIp\\TeemIp\\Extension\\IPv6Management\\Model\\_IPv6Address::GetName() should be compatible with DBObject::GetName($sType = Combodo\\iTop\\Core\\MetaModel\\FriendlyNameType::SHORT) in /var/www/html/env-production/teemip-ipv6-mgmt/src/Model/_IPv6Address.php on line 32, referer: http://localhost:8001/setup/wizard.php
[Wed Nov 03 20:37:47.726109 2021] [php7:notice] [pid 582] [client 172.17.0.1:55214] PHP Notice:  Uninitialized string offset: 0 in /var/www/html/setup/modulediscovery.class.inc.php(466) : eval()'d code on line 129, referer: http://localhost:8001/setup/wizard.php
```

**Comment by xtophe38** [comment]
Hum, indeed, GetName prototype has changed... and, furthermore, will be final at DBObject level in 3.1 !! I need to find a workaround. Thanks for the heads up.

**Comment by xtophe38** [comment]
Problem has been corrected with commit [d0b0d12](https://github.com/TeemIp/teemip-core-ip-mgmt/commit/d0b0d122783ad54fc9ddfc3544beaadd7c393e3c#diff-f4515ed0b2f3187a6022710d229a5bfa643be4e872f754e6177bf9b2e3e2daad)
